### DISCUSSION TITLE
Check custom image

### DISCUSSION THREAD

**Discussion by ajpanton** [getwud/wud#515] [Q&A]
I have a custom image, but it's pretty much a vanilla php one, just with some database extensions installed. The dockerfile is just two lines, which I add directly to the docker compose:

```
services:
  php:
    build:
      context: .
      dockerfile_inline: |
        FROM php:8-fpm-alpine
        RUN apk add --no-cache postgresql-dev && docker-php-ext-install pdo_pgsql
    ...
```
Is there any way to have WUD notify me when there's a new php version available? I'm not really sure how it would look for it though. Some tool to add the original php digest somewhere for WUD to read it? Use a label with the complete version number (would probably want to change that FROM command as well to keep track of what's being used)?

**Comment by N1k145** [comment]
Hey were you able to find a solution?

This is also a feature that I am missing from all solutions Iooked into

**Comment by viharm** [comment]
I have the exact same requirement.

Although not a direct solution, I'm wondering if this workaround would work...

Would it be worth having a _PHP_ container sitting & doing nothing... not even running, with it's `wud`... tags configured for watching PHP? And then forcing `wud.watch` to `true` ? I'm going to have to force `wud.watch` to `false` for my custom PHP image.

I'm hoping a proper solution comes along, but this might work.

**Comment by thecoshman** [comment]
I think you're best option here is to actually publish the image somewhere, like ghcr, then have a regular job that can use that instance of the container to run a command to check if the version in the container can be updated. Something like `... exec php -version` to get the version of PHP in the container, but also install latest PHP in the job and get the version there, if they job installed a newer version, trigger rebuild of the the image.
With a new image built, WUD can detect a new version is available, even if it's just tagged `:latest`, by comparing the sha
